predicators/envs/burger.py

```diff
@@ -610,8 +642,9 @@ def render_state_plt(
         # The DPI has to be sufficiently high otherwise when the matplotlib
         # figure gets converted to a PIL image, text in the image can become
         # blurry.
-        fig, ax = plt.subplots(1, 1, figsize=figsize, dpi=200)
-        plt.suptitle(caption, wrap=True)
+        fig, ax = plt.subplots(1, 1, figsize=figsize, dpi=216)
+        # plt.suptitle(caption, wrap=True)
```

Comment:
why did you take out captioning?

Reply:
I don't remember... I think maybe when I was doing the cropped images. I want the matplotlib output to be the same dimension all the time and not have much else other than the grid itself (no white padding!) so that I can basically hardcode some cell-width values (so it makes it more reliable to automatically get a crop of a particular set of cells). 

I think maybe the caption was unnecessary and adds variable whitespace which can mess up the cropping. We aren't going to use the caption at all at least when creating demonstrations to train on.